apps/frontend/src/components/Class/Overview/index.tsx

```diff
@@ -1,24 +1,212 @@
+import {
+  QuestionMark,
+  User,
+  UserXmark,
+  VideoCamera,
+  VideoCameraOff,
+} from "iconoir-react";
+import _ from "lodash";
+import { Link } from "react-router-dom";
+
+import {
+  CONSENSUS_THRESHOLD,
+  METRIC_MAPPINGS,
+  MINIMUM_RESPONSES_THRESHOLD,
+  MetricName,
+} from "@repo/shared";
+
 import Details from "@/components/Details";
+import CourseContext from "@/contexts/CourseContext";
+import { useReadUser } from "@/hooks/api";
 import useClass from "@/hooks/useClass";
+import useCourse from "@/hooks/useCourse";
+import { signIn } from "@/lib/api";
+import { ICourse } from "@/lib/api";
 
 import styles from "./Overview.module.scss";
 
+enum Consensus {
+  Yes = "Yes",
+  No = "No",
+  BellowThreshold = "BellowThreshold",
+  Indeterminate = "Indeterminate",
+}
+
 export default function Overview() {
   const { class: _class } = useClass();
+  return (
+    <CourseContext.Provider value={{ course: _class.course }}>
```

Comment:
The `useClass` already has a `course` field, so the additional `CourseContext` here is unnecessary.